lib/telemetry.js

```diff
@@ -60,5 +66,30 @@ export const startPingLoop = () => {
     point.tag('arch', arch())
     point.tag('deployment_type', DEPLOYMENT_TYPE)
     writeClient.writePoint(point)
-  }, 10 * 60 * 1000)
+    writeClient.flush().catch(handleFlushError)
+    await timers.setTimeout(10 * 60 * 1000) // 10 minutes
+  }
+}
+
+export const runMachinesLoop = async () => {
+  while (true) {
+    const point = new Point('machine')
+    point.intField('cpu_count', cpus().length)
+    point.intField('cpu_speed_mhz', cpus()[0].speed)
+    const model = cpus()[0].model.toLowerCase()
+    point.tag(
+      'cpu_brand',
+      model.includes('intel')
+        ? 'intel'
+        : model.includes('amd')
+          ? 'amd'
+          : model.includes('apple')
+            ? 'apple'
+            : 'unknown'
+    )
+    point.intField('memory_total_b', totalmem())
+    writeClientMachines.writePoint(point)
+    writeClientMachines.flush().catch(handleFlushError)
+    await timers.setTimeout(86_400) // 1 day
```

Comment:
I think this is 1/1000 of a day if `setTimeout` accepts the interval in milliseconds.

Do you know if the implementation in Node.js tracks wall time or process time? When my computer sleeps for 8 hours during the night, will the timer keep progressing, or will it be delayed by that 8 hours?

Maybe we can submit the machine stats more often, e.g. every 4 hours or every 6 hours?